### DISCUSSION TITLE
WebAuthn support

### DISCUSSION THREAD

**Discussion by girlbossdev** [pufferpanel/pufferpanel#1202] [Ideas]
It would be pretty nice if we could use it for 2FA, apart from the added security it's also a lot more convenient than having to open the app and type a code every time you log in

**Comment by nepcore** [comment]
I did look at WebAuthn back when I implemented TOTP, but it's much less straight forward with little info on it besides the spec itself which I've not yet had time to work my way through properly  

If someone wants to contribute such a feature, feel free to reach out to us (preferably on our discord), on my to do list it's currently honestly quite low though as TOTP is there and there are a lot of things I want to do that affect more users with less effort
